app/internal/mail.py

```diff
@@ -0,0 +1,65 @@
+import os
```

Comment:
Can't see why this shouldn't be in the same subpackage as the rest of the feature's code.

Reply:
in a different pull request I saw that Yam told him to split the route related code and the complicated stuff to seperate files
so this is what I did here

Reply:
ok